### PR TITLE
[JS] Topic: Loops #492

### PR DESCRIPTION
**Pull request by gurpreet-legend**
<hr>

## Description 📜
Created tutorial for Loops with proper documentation

Fixes #492  

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [ ] Android Dev (Flutter, Java, Kotlin)
- [ ] Backend Dev (Java, .NET, Python)
- [ ] C/CPP
- [ ] Competitive Programming
- [ ] Cyber Security
- [ ] DSA
- [ ] Database
- [ ] Datascience with Python
- [ ] Datascience with R
- [ ] Frontend Dev (HTML, CSS, JS)
- [ ] Frontend Dev (React, Angular, Vue)
- [ ] Golang
- [ ] Interview Prep
- [ ] Java
- [x] JavaScript
- [ ] MERN
- [ ] Machine Learning
- [ ] Open Source
- [ ] Python
- [ ] Rust
- [ ] Statistics
- [ ] UI/UX

<hr>
 
## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

### CODE REVIEW

**Comment by paramsiddharth** [comment]
@Creatoon @dakshsethi Please review.

**Comment by s4-2** [comment]
Hii @gurpreet-legend , Good job. The explaination is really good and implementation of code is simple and good. But I just want you to let you know that there is minor space gap in the description. There should be no space between # and 492 --> write as(#492), so it automatically give the link of the issue which you have solved.
Hope you understand, feel free to ask if you have any doubt:)

**Comment by gurpreet-legend** [comment]
I have corrected it thanks to you @s4-2 :)

**Comment by paramsiddharth** [comment]
> I have corrected it thanks to you @s4-2 :)

Please push the update to the branch, so that it may reflect in the PR.

**Comment by paramsiddharth** [comment]
@Creatoon @dakshsethi Please review.

**Comment by gurpreet-legend** [comment]
> > I have corrected it thanks to you @s4-2 :)
> 
> Please push the update to the branch, so that it may reflect in the PR.

It was just an error in the initial comment, nothing updated in the code itself, waiting for the reviews from mentors.

**Review by dakshsethi — APPROVED** [review]

File: Web_Development/Javascript/Topics/5.Syntax/2.Loops/README.md
Comment (dakshsethi) [inline Web_Development/Javascript/Topics/5.Syntax/2.Loops/README.md:237]: Add names of both the contributors with the link to their GitHub profiles like this:
**Contributors:** [Harsh Mathur](https://github.com/harshmathurx) and [Gurpreet Singh](https://github.com/gurpreet-legend)
`**Contributors:** [Harsh Mathur](https://github.com/harshmathurx) and [Gurpreet Singh](https://github.com/gurpreet-legend)`

**Review by dakshsethi — CHANGES_REQUESTED** [review]

**Comment by paramsiddharth** [comment]
@Creatoon @dakshsethi @s4-2 Please review.

**Review by s4-2 — APPROVED** [review]
Looks good to me:)

File: Web_Development/Javascript/Topics/5.Syntax/2.Loops/README.md
Comment (gurpreet-legend) [reply Web_Development/Javascript/Topics/5.Syntax/2.Loops/README.md:237]: Sure sir, but can I ask who is Harsh Mathur ?

Comment (dakshsethi) [reply Web_Development/Javascript/Topics/5.Syntax/2.Loops/README.md:237]: > Sure sir, but can I ask who is Harsh Mathur ?

He is the original creator of the file.

Comment (gurpreet-legend) [reply Web_Development/Javascript/Topics/5.Syntax/2.Loops/README.md:237]: Ok sir done

**Comment by gurpreet-legend** [comment]
@paramsiddharth you can review my PR, it has been reviewed

**Review by dakshsethi — APPROVED** [review]
Looks good to me!! 🔥🔥🔥

File: Web_Development/Javascript/Topics/5.Syntax/2.Loops/README.md
Comment (Creatoon) [inline Web_Development/Javascript/Topics/5.Syntax/2.Loops/README.md:118]: Also write its proof, if you are claiming something

File: Web_Development/Javascript/Topics/5.Syntax/2.Loops/README.md
Comment (Creatoon) [inline Web_Development/Javascript/Topics/5.Syntax/2.Loops/README.md:123]:
```suggestion
Syntax of ***for of*** loop :
```

File: Web_Development/Javascript/Topics/5.Syntax/2.Loops/README.md
Comment (Creatoon) [inline Web_Development/Javascript/Topics/5.Syntax/2.Loops/README.md:149]: Give example

File: Web_Development/Javascript/Topics/5.Syntax/2.Loops/README.md
Comment (Creatoon) [inline Web_Development/Javascript/Topics/5.Syntax/2.Loops/README.md:213]:
```suggestion
> You can iterate over an array with the help of ``Array.forEach()`` method as well instead of using loops. It was introduced with **ES5** version.
```

File: Web_Development/Javascript/Topics/5.Syntax/2.Loops/README.md
Comment (Creatoon) [inline Web_Development/Javascript/Topics/5.Syntax/2.Loops/README.md:217]:
```suggestion
>array.forEach(function(currentValue, index))
```

File: Web_Development/Javascript/Topics/5.Syntax/2.Loops/README.md
Comment (Creatoon) [inline Web_Development/Javascript/Topics/5.Syntax/2.Loops/README.md:219]:
```suggestion
>- *currentValue* - Value of the current element
>- *index* - Index of the current element
```

**Review by Creatoon — CHANGES_REQUESTED** [review]

**Comment by paramsiddharth** [comment]
@gurpreet-legend Please make the abovementioned changes requested by Raj.

File: Web_Development/Javascript/Topics/5.Syntax/2.Loops/README.md
Comment (gurpreet-legend) [reply Web_Development/Javascript/Topics/5.Syntax/2.Loops/README.md:118]: Done sir :)

File: Web_Development/Javascript/Topics/5.Syntax/2.Loops/README.md
Comment (gurpreet-legend) [reply Web_Development/Javascript/Topics/5.Syntax/2.Loops/README.md]: Done sir :)

**Review by gurpreet-legend — COMMENTED** [review]
@Creatoon I have done all the changes requested by you, please review :)

**Review by Creatoon — APPROVED** [review]

**Comment by gurpreet-legend** [comment]
> @gurpreet-legend Please make the abovementioned changes requested by Raj.

@paramsiddharth , I have done all the changes sir :)

**Review by paramsiddharth — APPROVED** [review]

**Pull request merged**
